fintech-examples/fintech-ui/Dockerfile

```diff
@@ -48,14 +48,14 @@ COPY ./nginx.conf.erb /etc/nginx/conf.d/default.conf.erb
 # copy entry.sh file
 COPY ./entry.sh /opt/entry.sh
 
-# set permissions
-RUN chgrp -R root /var/cache/nginx && \
-    find /var/cache/nginx -type d -exec chmod 775 {} \; && \
-    find /var/cache/nginx -type f -exec chmod 664 {} \; && \
-    chmod 775 /var/run && \
+# support running as arbitrary user which belogs to the root group
+RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx && \
     chmod 775 /opt/entry.sh && \
```

Comment:
not related to this PR, but I'm wondering why do we need to set so globally open permissions?